client/X11/xf_input.c

```diff
@@ -142,25 +143,30 @@ static BOOL register_input_events(xfContext* xfc, Window window)
 					{
 						double max_pressure = t->max;
 
-						if (strstr(dev->name, "Stylus Pen") || strstr(dev->name, "Pen Pen"))
+						char* devName = _strdup(dev->name);
```

Comment:
Here `CharLowerBuffA` might do the convertion to lower case for you. Also I guess that the devName always has a reasonable size (max 100 ?) and so perhaps we could avoid the dynamic allocation and have `char devName[200];` ?